eng/Versions.props

```diff
@@ -23,5 +23,6 @@
     <MicrosoftDotNetBuildTasksPackagingPackageVersion>1.0.0-beta.18613.4</MicrosoftDotNetBuildTasksPackagingPackageVersion>
     <MicrosoftDotNetCoreFxTestingPackageVersion>1.0.0-beta.18613.4</MicrosoftDotNetCoreFxTestingPackageVersion>
     <MicrosoftDotNetBuildTasksConfigurationPackageVersion>1.0.0-beta.18613.4</MicrosoftDotNetBuildTasksConfigurationPackageVersion>
+    <MicrosoftDotNetBuildTasksFeedPackageVersion>2.2.0-beta.18613.3</MicrosoftDotNetBuildTasksFeedPackageVersion>
```

Comment:
I was just taking a look around (seeing how you implemented publishing) and noticed that you have dependencies on Arcade Tasks.Feed and HelixSDK but no corresponding entry in `eng\Version.Details.xml`. Is the goal to update them manually? IDK a lot about Maestro, though. @jcagme

Reply:
No, the goal is to use DARC to update it, I didn't add it yet as I have more cleanups to do. I will add it in a follow up PR as I need to add symbol publishing and delete dependencies to the OLD package coming from buildtools.